src/pybind/mgr/dashboard/frontend/cypress/e2e/common/global.feature.po.ts

```diff
@@ -25,6 +25,24 @@ When('I click on {string} button', (button: string) => {
   cy.get(`[aria-label="${button}"]`).first().click({ force: true });
 });
 
+/**
+ * General purpose click on function
+ * @param dataCy data-cy of the element to click on
+ */
+And('I click on {string}', (dataCy: string) => {
+  cy.get(`[data-cy=${dataCy}]`).click();
```

Comment:
There is some inconsistency in naming the cypress test IDs, there are `data-testid` which is most used in code and there is also `data-cy` which is also used but very less.
Lets fix and use `data-testid` only at all for now. https://tracker.ceph.com/issues/70986

Reply:
Sure, I've change it. 

I was following cypress best practices that indicate `data-*` as variables to use

https://docs.cypress.io/app/core-concepts/best-practices

Also `data-cy`, `data-test` and `data-testid` are indicated as examples for bestpractices as far as I see